xml/art_amd-sev.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+  <!ENTITY productnamex "&sls;">
+  <!ENTITY productnumberx "15">
+]>
+
+<article xml:id="art.amd-sev" xml:lang="en" version="5.0"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>AMD Secure Encrypted Virtualization (AMD-SEV) Guide</title>
+ <info>
+  <productname>&productnamex;</productname>
+  <productnumber>&productnumberx;</productnumber>
+  <date><?dbtimestamp format="B d, Y" ?></date>
+  <!-- abstract: -->
+  <xi:include xpointer="element(/1/2/1)" href="art_sles_amd-sev.xml"/>
+ </info>
+ <!-- intro paras: -->
+ <xi:include xpointer="element(/1/3)" href="art_sles_amd-sev.xml"/>
+ 
+ <!-- sections: -->
+ 
+ <xi:include xpointer="element(/1/4)" href="art_sles_amd-sev.xml"/>
+ <xi:include xpointer="element(/1/5)" href="art_sles_amd-sev.xml"/>
+ <xi:include xpointer="element(/1/6)" href="art_sles_amd-sev.xml"/>
+ <xi:include xpointer="element(/1/7)" href="art_sles_amd-sev.xml"/>
+ <xi:include xpointer="element(/1/8)" href="art_sles_amd-sev.xml"/>
+ 
+ <xi:include href="common_gfdl1.2_i.xml"/>
```

Comment:
The xincludes look odd. I would make it a single file and merge art_sles_amd-sev.xml here

Reply:
It looked odd to me, too, but you suggested starting with the PCIDSS article, and that did it, so I copied the structure directly. I do not know why the PCIDSS article does it that way.

I will combine the docs _after_ I have addressed the other points, to keep to one step at a time.

Reply:
Sorry, bad choice from my side. I did not had a closer look at the PCIDSS guide. I assume it was done that way in order to use it as a stand-alone document and to integrate it in one of our guides at the same time.

Reply:
Aha! No problem. If something looks weird, I tend to leave it alone on the basis that there is probably a good reason for it that I don't yet understand.

I have a few more suggestions to incorporate, then I will merge the 2 docs into one.